src/components/RestModules/ModuleCard.js

```diff
@@ -1,15 +1,14 @@
 import React from 'react';
-import { Card } from 'antd';
 
 const ModuleCard = props => {
-  const { module } = props;
   return (
     <>
-      <Card title={module.modulename}>
-        <h3>Description: {module.moduledescription}</h3>
-        {/* PLEASE DO NOT REMOVE THE FOLLOWING LINE */}
-        {props.children}
-      </Card>
+      {/* <Card title={module.modulename}>
+        <h3>Description: {module.moduledescription}</h3> */}
+      {/* PLEASE DO NOT REMOVE THE FOLLOWING LINE */}
+      {props.children}
```

Comment:
and if this component now ONLY renders `{props.children}` in a `<React.Fragment />` then it should ~~probably~~ definitely not be a component anymore. just taking up space for no reason